### PR TITLE
Fix Swagger Linting Error for Microsoft.Features

### PR DESCRIPTION
**Pull request by zjpjack**
Fix Swagger Linting Error for Microsoft.Features

### CODE REVIEW

**Comment by annatisch** [comment]
Thanks @zjpjack - your new examples have a couple of model validation errors - could you take a look?
```
 error: 
operationId: ListOperations
scenario: List Features operations
source: response
responseCode: '200'
severity: 0
errorCode: OBJECT_ADDITIONAL_PROPERTIES
errorDetails:
  code: OBJECT_ADDITIONAL_PROPERTIES
  params:
    - - properties
  message: 'Additional properties not allowed: properties'
  path: value/0/display
  title: /definitions/generated.nested.definitions.Operation.properties.display
  description: The object that represents the operation.
  similarPaths:
    - value/1/display
  position:
    line: 286
    column: 20
  url: >-
    specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json
  jsonPosition:
    line: 12
    column: 36
  jsonUrl: >-
    specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/examples/listFeaturesOperations.json

 error: 
operationId: Features_Get
scenario: Get feature
source: response
responseCode: '200'
severity: 0
errorCode: OBJECT_ADDITIONAL_PROPERTIES
errorDetails:
  code: OBJECT_ADDITIONAL_PROPERTIES
  params:
    - - type
      - id
      - properties
      - name
  message: 'Additional properties not allowed: type,id,properties,name'
  path: properties
  title: /definitions/FeatureProperties
  description: Information about feature.
  position:
    line: 232
    column: 26
  url: >-
    specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json
  jsonPosition:
    line: 13
    column: 21
  jsonUrl: >-
    specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/examples/getFeature.json

 error: 
operationId: Features_Register
scenario: Register feature
source: response
responseCode: '200'
severity: 0
errorCode: OBJECT_ADDITIONAL_PROPERTIES
errorDetails:
  code: OBJECT_ADDITIONAL_PROPERTIES
  params:
    - - type
      - id
      - properties
      - name
  message: 'Additional properties not allowed: type,id,properties,name'
  path: properties
  title: /definitions/FeatureProperties
  description: Information about feature.
  position:
    line: 232
    column: 26
  url: >-
    specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json
  jsonPosition:
    line: 13
    column: 21
  jsonUrl: >-
    specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/examples/registerFeature.json
```

**Comment by zjpjack** [comment]
@annatisch Can I check these model validation on my local machine?

**Comment by annatisch** [comment]
I believe so - you need to set up the npm environment (specifically the oav package).
This is the script we run:
https://github.com/Azure/azure-rest-api-specs/blob/master/scripts/modelValidation.js

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json [hunk 1]
```diff
@@ -224,6 +274,49 @@
...
+              "description": "Service provider: Microsoft.Features",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Profile, endpoint, etc.",
+              "type": "string"
+            },
+            "operation": {
+              "description": "Operation type: Read, write, delete, etc.",
+              "type": "string"
```
Comment (annatisch) [inline specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: Should this be an enum of specific values?

File: specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json [hunk 2]
```diff
@@ -224,6 +274,49 @@
...
+        },
+        "display": {
+          "description": "The object that represents the operation.",
+          "properties": {
+            "provider": {
+              "description": "Service provider: Microsoft.Features",
+              "type": "string"
+            },
+            "resource": {
+              "description": "Resource on which the operation is performed: Profile, endpoint, etc.",
```
Comment (annatisch) [inline specification/resources/resource-manager/Microsoft.Features/stable/2015-12-01/features.json]: Should this be an enum of specific values? Or is it arbitrary depending on the feature?

**Comment by annatisch** [comment]
ping @zjpjack - did you have any comments on my review feedback?

**Comment by zjpjack** [comment]
@annatisch I replied you last Friday, maybe you didn't notice. I am waiting for your response. Thanks

**Comment by annatisch** [comment]
Thanks @zjpjack - I'm confused - you responded to my question about enums? In email or here - I can't see it :)

**Comment by zjpjack** [comment]
@annatisch please choose "File changes" tab. Your comments are in there, and I also reply u there

**Comment by annatisch** [comment]
@zjpjack - weird - I can't see any reply to my comments in the file changes either... Maybe you could just paste your responses here in the main conversation?

**Comment by zjpjack** [comment]
@annatisch Now you should see them :)

**Review by annatisch — APPROVED** [review]

**Pull request merged**